cli/test/support/testData.ts

```diff
@@ -0,0 +1,85 @@
+import { csvw, hydra, rdf, schema } from '@tpluscode/rdf-ns-builders'
```

Comment:
This is an interesting part, where I put some test project and other necessary resources to the triple store